README.md

```diff
@@ -92,10 +92,10 @@ bin/proguard.sh @myconfig.pro
 or on Windows:
 
 ```
-bin\proguard @myconfig.pro
+bin\proguard.bat @myconfig.pro
 ```
 
-All available options are described in the [Configuration section](../configuration/usage.md).
+All available options are described in the [Configuration section](docs/md/manual/configuration/usage.md).
```

Comment:
```suggestion
All available options are described in the [configuration section of the manual](https://www.guardsquare.com/manual/configuration/usage).
```

This can actually link to the published manual instead.